app/views/shared/context_tables/_heading.html.erb

```diff
@@ -1,24 +1,31 @@
-<table class="govuk-table">
-  <tbody class="govuk-table__body">
-    <tr class="govuk-table__row">
-      <th scope="row" class="govuk-table__header govuk-!-width-one-quarter">Heading</th>
-      <td class="govuk-table__cell"><%= @heading.short_code %></td>
-      <td class="govuk-table__cell"/>
-    </tr>
-    <tr class="govuk-table__row">
-      <th scope="row" class="govuk-table__header">Classification</th>
-      <td class="govuk-table__cell"><%= @heading.to_s %></td>
-      <td class="govuk-table__cell"/>
-    </tr>
-    <% if @heading.declarable? %>
-      <%= render 'shared/context_tables/supplementary_unit_row', supplementary_unit: @supplementary_unit %>
-    <% end %>
-    <tr class="govuk-table__row">
-      <th scope="row" class="govuk-table__header">Date of trade</th>
-      <td class="govuk-table__cell">
-        <%= @search.date.to_formatted_s(:long) %>
-      </td>
-      <td class="govuk-table__cell"></td>
-    </tr>
-  </tbody>
-</table>
+<dl class="govuk-summary-list">
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">
+      Heading
+    </dt>
+    <dd class="govuk-summary-list__value">
+      <%= @heading.short_code %>
+    </dd>
+  </div>
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">
+      Classification
+    </dt>
+    <dd class="govuk-summary-list__value">
+      <%= @heading.to_s %>
+    </dd>
+  </div>
+  <% if @heading.declarable? %>
+    <%= render 'shared/context_tables/supplementary_unit_row', supplementary_unit: @supplementary_unit %>
+  <% end %>
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">
+      Date of trade
+    </dt>
+    <dd class="govuk-summary-list__value">
+      <%= @search.date.to_formatted_s(:long) %>
+    </dd>
+    <dd class="govuk-summary-list__actions">
```

Comment:
Should the actions section be here? Its not present on the other rows?

Reply:
I'll tidy up in a bit. I do need the actions on another branch, yeah